trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/wave_drag_volume.py

```diff
@@ -41,6 +42,16 @@ def wave_drag_volume(vehicle,mach,scaling_factor):
             num_main_wings += 1
         if num_main_wings > 1:
             raise NotImplementedError('This function is not designed to handle multiple main wings.')
+    
+    main_wing = vehicle.wings.main_wing
+    # estimation of leading edge sweep if not defined 
+    if main_wing.sweeps.leading_edge == None:                                                     
+        QC_sweep                       = main_wing.sweeps.quarter_chord
```

Comment:
there's a function in suave for this already you could just call

Reply:
Actually it is computed many placed but there is no one-off function

Reply:
Thanks!